Header header
goal_detection/Goal[] goals
bool valid
